cmd/accelctl/main.go

```diff
@@ -118,6 +120,37 @@ func main() {
 					},
 				},
 			},
+			{
+				Name:  "convert",
+				Usage: "Convert an image locally (daemonless)",
+				Flags: []cli.Flag{
+					&cli.StringFlag{Name: "config", Required: true, Usage: "Specify the path of config in yaml format"},
+				},
+				ArgsUsage: "[SOURCE]",
+				Action: func(c *cli.Context) error {
+					source := c.Args().First()
+					if source == "" {
+						return fmt.Errorf("source is required")
+					}
+
+					cfg, err := config.Parse(c.String("config"))
+					if err != nil {
+						return err
+					}
+
+					handler, err := handler.NewLocalHandler(cfg)
+					if err != nil {
+						return err
+					}
+
+					err = handler.Convert(c.Context, source, true)
+					if err != nil {
```

Comment:
It is not necessary to check err as we return it anyway.

Reply:
Thanks, fixed.